simplesat/tests/test_rules_generator.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import io
+import unittest
+
+from enstaller.new_solver import Pool
+
+from ..rules_generator import RuleType, RulesGenerator
+from ..test_utils import Scenario
+
+
+class TestRulesGenerator(unittest.TestCase):
+
+    def test_prefer_installed(self):
+
```

Comment:
At first I was going to whine, but when I rewrote it with sections it became much nicer. :grinning: